prediction/src/algorithms/segment/trained_model.py

```diff
@@ -2,30 +2,29 @@
 """
     algorithms.segment.trained_model
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
```

Comment:
Please try and avoid making whitespace changes that are unrelated to the commit message contents. Feel free to make them, but separating them out into their own commit makes it much easier to review stuff!

Reply:
Thanks for noted that. I will stick with it further.